src/musicbrainz/chromaprinter.cpp

```diff
@@ -12,6 +12,16 @@
 
 namespace
 {
+// Type declarations of *fprint and *encoded pointers need to account for Chromaprint API version
+// (void* -> uint32_t*) and (void* -> char*) changed in versions v1.4.0 or later -- alyptik 12/2016
+#if (CHROMAPRINT_VERSION_MINOR > 3) || (CHROMAPRINT_VERSION_MAJOR > 1)
+    typedef uint32_t* uint32_p;
+    typedef char* char_p;
```

Comment:
well, my comment was about the typedef lines, not the #if
you can use the same indentation as https://github.com/mixxxdj/mixxx/blob/master/src/defs_version.h#L52